pkg/network/protocols/http/monitor.go

```diff
@@ -11,10 +11,11 @@ package http
 import (
 	"errors"
 	"fmt"
-	"github.com/DataDog/datadog-agent/pkg/network/ebpf/probes"
 	"syscall"
 	"unsafe"
 
+	"github.com/DataDog/datadog-agent/pkg/network/ebpf/probes"
```

Comment:
let's move it to the last section of imports (the one that contains the agent repo imports)